web/server/api/endpoints/models.py

```diff
@@ -25,14 +25,16 @@ def get_models(
     """Get a mapping of model names to model metadata"""
     output = []
 
-    def _get_model_type(model: Model) -> str | None:
+    context.load()
```

Comment:
I asked a few times in another PR so excuse the repetitiveness, but won't the file watcher already have reloaded the context?